WorkloadManagementSystem/Service/JobMonitoringHandler.py

```diff
@@ -242,6 +242,14 @@ def export_getJobLoggingInfo( jobID ):
     return gJobLoggingDB.getJobLoggingInfo( jobID )
 
 ##############################################################################
+  types_getJobsParameters = [ ListType, ListType ]
+  @staticmethod
+  def export_getJobsParameters ( jobIDs, parameters ):
+    if not ( jobIDs and parameters ) : 
+      return S_OK( {} )
+    return gJobDB.getAttributesForJobList( jobIDs, parameters )
+
+##############################################################################
```

Comment:
This is fine. Just one question - I think that getJobParameters() is better english, but I am not a native speaker. Ideal names in the code should read as normal language phrases